samples/peripheral/802154_sniffer/prj.conf

```diff
@@ -0,0 +1,27 @@
+# Enabling radio driver
+CONFIG_NETWORKING=y
+CONFIG_IEEE802154=y
+CONFIG_IEEE802154_RAW_MODE=y
+CONFIG_NET_PKT_TIMESTAMP=y
+
+# Allow sharing the RTC between IEEE 802.15.4 and Zephyr
+CONFIG_NRF_RTC_TIMER_USER_CHAN_COUNT=3
+
+# Shell configuration
+CONFIG_SHELL=y
+CONFIG_SHELL_PROMPT_UART=""
+CONFIG_SHELL_VT100_COLORS=n
+
+# LED indication
+CONFIG_DK_LIBRARY=y
+
+# USB configuration
+CONFIG_USB_DEVICE_STACK=y
+CONFIG_USB_DEVICE_MANUFACTURER="Nordic Semiconductor ASA"
+CONFIG_USB_DEVICE_PRODUCT="nRF 802154 Sniffer"
+CONFIG_USB_DEVICE_VID=0x1915
+CONFIG_USB_DEVICE_PID=0x154a
```

Comment:
Is command line compatible with the previous sniffer? If not probably 0x154b would be better for extcap scripts to be able to distinguish when connecting

Reply:
It's nearly indistinguishable. Unfortunately the zephyr shell appends more newline characters than the previous sniffer did. This means that the extcap script has to be modified slightly to either align with the intricacies of this sample, or make such a modification on the extcap side that it can handle both this version and the previous one.

Reply:
Ok, so I think it's fine to keep 0x154a